utils/passkeys/load-client-from-passkey.ts

```diff
@@ -0,0 +1,71 @@
+import { RPID } from "./passkeys.constants";
+import { createTurnkeyAccountFromWalletId } from "./create-turnkey-account";
+import { TurnkeyStoreInfo } from "./passkeys.interfaces";
+import { PasskeyStamper } from "@turnkey/react-native-passkey-stamper";
+import { TurnkeyClient } from "@turnkey/http";
+import { setTurnkeyClient } from "./turnkey-clients";
+import config from "@/config";
+
+type LoadAccountFromPasskeyParams = {
+  setStatusString: (statusString: string) => void;
+  setPreviousPasskeyName: (passkeyName: string) => void;
+  setTurnkeyInfo: (turnkeyInfo: TurnkeyStoreInfo) => void;
+};
+
+export const loadAccountFromPasskey = async ({
```

Comment:
fine if this is just a poc, but is this pattern of passing side effects to what are essentially api calls something we've already established or was that just to move quickly?

The more side effects we have spread out, the harder the logic is to follow.